modules/nf-core/survivor/merge/main.nf

```diff
@@ -0,0 +1,51 @@
+process SURVIVOR_MERGE {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::survivor=1.0.7" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/survivor:1.0.7--h9a82719_1':
+        'quay.io/biocontainers/survivor:1.0.7--h9a82719_1' }"
+
+    input:
+    tuple val(meta), path(vcfs)
+
+    output:
+    tuple val(meta), path("*.vcf")   , emit: vcf
+    path "versions.yml"              , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: [:]
```

Comment:
I was confused by the need to have args as a map- but I take it this is because SURVIVOR has positional arguments and you need to ensure the order?

I think if these are compulsory non-file arguments then we'd probably be better to formulate these as channel inputs (see https://github.com/nf-core/nf-co.re/pull/1486), assuming those are the accepted new guidelines.

Reply:
These also state that if there are defaults, they should be able to be overwritten by via `args`. I think that would be the best here. I'll add the defaults here